module/VuFind/tests/integration-tests/src/VuFindTest/Mink/HoldingsTest.php

```diff
@@ -82,17 +82,32 @@ function ($a) {
             $set
         );
 
-        return [...$msgSet, ...$groupSet, ...$allSet];
+        $totalSet = array_map(
```

Comment:
Would it be easier to add default values to the $loadBatchWise and $loadObservableOnly parameters to the test methods instead of doing this array remapping, or is there a reason we have to do it this way?

Reply:
Done